client/src/pages/course/[taskDetails].tsx

```diff
@@ -0,0 +1,294 @@
+import React, { useMemo, useState } from 'react';
+
+import { Header } from 'components';
+import { useRouter } from 'next/router'
+import { useAsync } from 'react-use';
+import {
+  Row,
+  Col,
+  Typography,
+  Tooltip,
+} from 'antd';
+
+import { EventTypeColor, EventTypeToName } from 'components/Schedule/model';
+
+import {
+  renderTag,
+} from 'components/Table';
+
+import withCourseData from 'components/withCourseData';
+import { withSession } from 'components';
+import { useLoading } from 'components/useLoading';
+
+import { CourseEvent, CourseService, CourseTaskDetails } from '../../services/course';
+import { CoursePageProps } from 'services/models';
+
+enum OccasionTypes {
+  UNKNOWN = 'UNKNOWN',
+  EVENT = 'EVENT',
+  TASK = 'TASK',
+}
+
+export function TaskDetailsPage(props: CoursePageProps) {
+  const router = useRouter();
+  const { taskDetails } = router.query;
```

Comment:
Let's rename the param to `eventDetails`

Reply:
no more need